webapp/src/ts/services/contact-save.service.ts

```diff
@@ -55,6 +56,10 @@ export class ContactSaveService {
     }
 
     const doc = this.prepare(submitted.doc);
+    if (typeof original === 'undefined' && doc.should_create_user === 'true') {
```

Comment:
```
typeof original === 'undefined'
```

This means we only perform this operation on new contacts.  Do we want that or should we support the possibility of adding a user when editing a contact? 

Also, I think we need to move this logic to the `prepare` function so that it gets run in any of the ways that a person is added via a contact form (e.g. if the person is created as a sibling doc at the same time the place is created or even if it is created in a repeat on the form).

Reply:
If we do support adding users while editing contacts, we need to make sure we _remove_ the `should_create_user` field here so that we do not keep creating users every time the contact is edited.